docs/faq.md

```diff
@@ -22,43 +22,37 @@ import TOCInline from "@theme/TOCInline"
 
 #### Semgrep Community Edition (CE)
 
-[Semgrep CE](https://github.com/semgrep/semgrep) is open source, licensed under [LGPL 2.1](https://tldrlegal.com/license/gnu-lesser-general-public-license-v2.1-(lgpl-2.1)) - you can use it at work, on private and proprietary code, no problem!
+[Semgrep CE](https://github.com/semgrep/semgrep) is a community-supported tool for code scanning. It's ideal for individuals, security auditors, and pentesters who need fast, one-off scans. You can use it at work, on private and proprietary code, no problem!
 
-Semgrep offers three paid products:
+Semgrep offers three paid products, designed for Application Security teams to use in production:
 
 * [<i class="fas fa-external-link fa-xs"></i> Semgrep Code](https://semgrep.dev/products/semgrep-code), a static application security testing (SAST) tool that can perform taint, cross-file, and cross-function analysis.
 * [<i class="fas fa-external-link fa-xs"></i> Semgrep Supply Chain](https://semgrep.dev/products/semgrep-supply-chain/), which performs dependency scanning.
 * [<i class="fas fa-external-link fa-xs"></i> Semgrep Secrets](https://semgrep.dev/products/semgrep-secrets/), which can detect and validate leaked secrets in code.
 
-#### Semgrep Registry
+#### Semgrep Rules
 
-The [<i class="fas fa-external-link fa-xs"></i> Semgrep Registry](https://semgrep.dev/explore) contains rules from different contributors.
+All Semgrep maintained rules are licensed under the [Semgrep Rules License v. 1.0](https://semgrep.dev/legal/rules-license/). The source for these rules is available at [semgrep/semgrep-rules](https://github.com/semgrep/semgrep-rules/).
 
-Semgrep **Community** rules written by the Semgrep team are licensed under the [Commons Clause](https://github.com/semgrep/semgrep-rules/blob/develop/LICENSE). The source for these Registry rules is available at [semgrep/semgrep-rules](https://github.com/semgrep/semgrep-rules/).
-
-Those rules licensed under the Commons Clause license cannot be resold without Semgrep, Inc. (“Semgrep”)’s permission. Since Semgrep offers a paid, hosted application, it’s important to have this restriction so other companies, like major cloud providers, cannot resell Semgrep’s rules as a competing service.
-
-In addition to Semgrep Community rules, [Semgrep Code](https://semgrep.dev/products/semgrep-code) includes **Pro rules** which are proprietary and only available to paying customers.
+These rules can only be used for internal business purposes. These rules cannot be resold without Semgrep, Inc. (“Semgrep”)’s permission. Since Semgrep offers a paid, hosted application, it’s important to have this restriction so other companies cannot resell Semgrep’s rules as a competing service. 
```

Comment:
```suggestion
These rules can only be used for internal business purposes. These rules cannot be resold without permission from Semgrep, Inc. (“Semgrep”). Since Semgrep offers a paid, hosted application, it’s important to have this restriction so other companies cannot resell Semgrep’s rules as a competing service. 
```

Reduced the special characters (`"` and `'`) in case a non-native speaker tries to feed the doc into a translator.